go.mod

```diff
@@ -56,7 +57,7 @@ require (
 	github.com/valyala/quicktemplate v1.1.1
 	golang.org/x/crypto v0.0.0-20190313024323-a1f597ede03a // indirect
 	golang.org/x/sys v0.0.0-20190312061237-fead79001313 // indirect
-	golang.org/x/tools v0.0.0-20190909030654-5b82db07426d
+	golang.org/x/tools v0.0.0-20190910044552-dd2b5c81c578
```

Comment:
can you do it in a separate pull request please? it will be easier to review and merge

Reply:
Yes, removed.

Reply:
Fixed.